state_handlers.js

```diff
@@ -193,6 +382,40 @@ function getTitlesHelper(stateObj) {
   );
 }
 
+function getTitleChunk(articleJson, stateObj) {
+  let articleLen = Object.keys(articleJson).length;
+  console.log('Length of array is: ' + articleLen);
+  let arrArticles = Object.keys(articleJson);
+  let retSpeech = '<break />';
+  let j = 0;
+  console.log(
+    'stateObj.attributes[titleCount] is: ' + stateObj.attributes['titleCount']
+  );
+  for (
+    let i = stateObj.attributes['titleCount'];
```

Comment:
I haven't taken a swing at an alternate implementation but this loop feels like it's more complex than it needs to be for the purpose of 'grab the next X titles'. Maybe a `slice()` and a `for(... of ...)` would simplify it and allow you to eliminate the multiple counters.